diff --git a/third_party/dawn/src/tint/utils/system/terminal_posix.cc b/third_party/dawn/src/tint/utils/system/terminal_posix.cc
index e820774..142e38c 100644
--- a/third_party/dawn/src/tint/utils/system/terminal_posix.cc
+++ b/third_party/dawn/src/tint/utils/system/terminal_posix.cc
@@ -38,6 +38,10 @@
 #include <string_view>
 #include <utility>
 
+// Extra includes to work with musl
+#include <sys/select.h>
+#include <sys/time.h>
+
 #include "src/tint/utils/containers/vector.h"
 #include "src/tint/utils/macros/compiler.h"
 #include "src/tint/utils/macros/defer.h"
