diff --git a/sys/v4l2/gstv4l2object.c b/sys/v4l2/gstv4l2object.c
index 5032da3..2be1a55 100644
--- a/sys/v4l2/gstv4l2object.c
+++ b/sys/v4l2/gstv4l2object.c
@@ -559,7 +559,7 @@ gst_v4l2_object_new (GstElement * element,
     v4l2object->fd_open = v4l2_fd_open;
     v4l2object->close = v4l2_close;
     v4l2object->dup = v4l2_dup;
-    v4l2object->ioctl = v4l2_ioctl;
+    v4l2object->ioctl = (int(*)(int, int, ...)) v4l2_ioctl;
     v4l2object->read = v4l2_read;
     v4l2object->mmap = v4l2_mmap;
     v4l2object->munmap = v4l2_munmap;
